docs/dataset.md

```diff
@@ -15,46 +15,64 @@ sd_hide_title: true
+    |-- systems  # Structure files for all systems (split by `two_char_code` and zipped)
+    |   |-- {two_char_code}.zip
+    |-- clusters # Pre-calculated cluster labels derived from the protein similarity dataset
+    |   |-- cluster=communities
+    |       |-- ...
+    |   |-- cluster=components
+    |       |-- ...
+    |-- splits # Split files and the configs used to generate them (if available)
+    |   |-- split.parquet
+    |   |-- split.yaml
+    |-- linked_structures # Apo and predicted structures linked to their holo systems
+    |   |-- {two_char_code}.zip
+    |-- links # Apo and predicted structures similarity to their holo structures
+    |   |-- apo_links.parquet
+    |   |-- pred_links.parquet
+
+--------------------------------------------------------------------------------
+                            miscellaneous data below
+--------------------------------------------------------------------------------
+
+    |-- dbs # TSVs containing the raw files and IDs in the foldseek and mmseqs sub-databases
     |   |-- subdbs
-    |       |-- {search_db}_{aln_type}
-    |           |-- ids.tsv
-    |-- entries
+    |       |-- apo.csv
+    |       |-- holo.csv
+    |       |-- pred.csv
+    |-- entries # Raw annotations prior to consolidation (split by `two_char_code` and zipped)
     |   |-- {two_char_code}.zip
-    |-- fingerprints
+    |-- fingerprints # Index mapping files for the ligand similarity dataset
     |   |-- ligands_per_inchikey.parquet
     |   |-- ligands_per_inchikey_ecfp4.npy
     |   |-- ligands_per_system.parquet
-    |-- index
-    |   |-- annotation_table.parquet
-    |   |-- annotation_table_nonredundant.parquet
-    |-- ligand_scores
-    |   |-- {fragment}.parquet
-    |-- ligands
-    |   |-- {fragment}.parquet
-    |-- mmp
+    |-- ligand_scores # Ligand similarity parquet dataset
+    |   |-- {hashid}.parquet
+    |-- ligands # Ligand data expanded from entries for computing similarity
+    |   |-- {hashid}.parquet
+    |-- mmp # Ligand matched molecular pairs (MMP) and series (MMS) data
     |   |-- plinder_mmp_series.parquet
     |   |-- plinder_mms.csv.gz
-    |-- scores
+    |-- scores # Protein similarity parquet dataset
     |   |-- search_db=apo
-    |   |   |-- {fragment}.parquet
+    |       |-- apo.parquet
     |   |-- search_db=holo
-    |       |-- {fragment}.parquet
+    |       |-- {chunck_id}.parquet
     |   |-- search_db=pred
-    |       |-- {fragment}.parquet
-    |-- splits
-    |   |-- split.parquet
-    |   |-- split.yaml
-    |-- systems
-        |-- {two_char_code}.zip
+    |       |-- pred.parquet
 ```
 
+:::{todo}
+- Add sections for missing tables
+- Fill missing elements into tables
+- Add descriptions for missing files
+:::
+
+We will describe the content of the `index`, `systems`, `clusters`, `splits`, `links` and `linked_structures` directories in detail below, the rest are described in the [miscellaneous section](#miscellaneous).
```

Comment:
For some reason implicit targets do not work in *MyST*, but we can use explicit ones instead (see review comment below)

```suggestion
We will describe the content of the `index`, `systems`, `clusters`, `splits`, `links` and `linked_structures` directories in detail below, the rest are described in the [miscellaneous section](#miscellaneous-target).
```